recipes/hdf5/all/conanfile.py

```diff
@@ -229,6 +229,7 @@ def _config_libname(lib):
 
             self.cpp_info.components[component_name].set_property("cmake_target_name", f"hdf5::{alias_target}")
             self.cpp_info.components[component_name].set_property("pkg_config_name", alias_target)
+            self.cpp_info.components[component_name].set_property("cmake_build_modules", [self._module_file_rel_path])
```

Comment:
```suggestion
```

no, CMakeDeps doesn't need this module file, it's a workaround for cmake_find_package

Reply:
I've had a go, please check